bench/ecu-send/sm_eval_send.c

```diff
@@ -141,6 +148,23 @@ void VULCAN_FUNC eval_mac(void)
 }
 #endif
 
+#ifdef BENCH_IAT
+void VULCAN_FUNC eval_iat(void)
+{
+    int i, len;
+
+    pr_progress("Measuring authentication frame timing");
+    for (i=0; i < 128; i++)
+    {
+        msg_id = -1;
+        do_send(&msp_ican, CAN_ID_PING, msg_ping, CAN_PAYLOAD_SIZE, /*block=*/1);
```

Comment:
this is all fine right? how about introducing some explicit nonce errors every x iterations? AFAIK right now the only test for that is in eval_demo but it would be good to have it here a bit more as well?